### ISSUE TITLE
GoToPose action feedback

### ISSUE THREAD

**Issue by bgromov** [jeguzzi/drone_arena_msgs#1]
Shouldn't the distance field in action feedback be `float32`?

https://github.com/jeguzzi/drone_arena_msgs/blob/bbfcd266c586bbecae5c3818148c6f67da837c8e/action/GoToPose.action#L8

Also, which distance exactly is that? 3D or 2D, i.e. does it ignore the Z coordinate?

**Comment by jeguzzi** [comment]
yes, who knows where this int32 comes from. Will change that. Thanks.

3D.
